### PR TITLE
feat: regenerate type and fix type error

### PR DESCRIPTION
**Pull request by BaBa0525**
## 📝 Description

This pull request involves changes to the `getWebsiteInfo` function in the `getWebsiteInfo.ts` file and the `paths` type in the `types.ts` file. The `getWebsiteInfo` function now allows for `null` values in the `pageParam` parameter and throws an error if the response data does not exist. The `/api/website/{website_id}` path in the `paths` type is replaced with `/` and the comment "Get Archived Dates" is removed.

### CODE REVIEW

**Comment by BaBa0525** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #19** <a href="https://app.graphite.dev/github/pr/GymSquad/admin-page/19?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #24** <a href="https://app.graphite.dev/github/pr/GymSquad/admin-page/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
      * **PR #25** <a href="https://app.graphite.dev/github/pr/GymSquad/admin-page/25?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/GymSquad/admin-page/24?utm_source=stack-comment).

File: src/api/getWebsiteInfo.ts [hunk 1]
```diff
@@ -19,16 +19,16 @@ export const getWebsiteInfo = async ({
...
   });
 
-  if (response.error) {
-    throw new Error(response.error.error);
+  if (response.data) {
+    return response.data;
   }
 
-  return response.data;
+  throw new Error("Fail to get website info");
```
Comment (BaBa0525) [inline src/api/getWebsiteInfo.ts:33]: 新的 error type 是 unknown，不太確定怎麼 handle 比較好 ouo

File: src/api/types.ts
Comment (BaBa0525) [inline src/api/types.ts:1]: Generated by openapi-typescript

File: src/api/getWebsiteInfo.ts (hunk 1 above)
Comment (alan910127) [reply src/api/getWebsiteInfo.ts:33]: 我要想辦法去把 error 的 type 用出來嗎，我可以研究一下

Comment (BaBa0525) [reply src/api/getWebsiteInfo.ts:33]: 都可以 不過不急 感覺可以加了再改

**Review by alan910127 — APPROVED** [review]

**Comment by BaBa0525** [comment]
### Merge activity

* **Feb 20, 3:51 AM EST**: @BaBa0525 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/GymSquad/admin-page/24).
* **Feb 20, 3:54 AM EST**: [Graphite](https://app.graphite.dev/github/pr/GymSquad/admin-page/24) rebased this pull request as part of a merge.
* **Feb 20, 3:55 AM EST**: @BaBa0525 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/GymSquad/admin-page/24).

**Pull request merged**